extensions/smallrye-reactive-messaging/deployment/src/test/java/io/quarkus/smallrye/reactivemessaging/blocking/beans/BeanReturningMessages.java

```diff
@@ -0,0 +1,34 @@
+package io.quarkus.smallrye.reactivemessaging.blocking.beans;
+
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import org.eclipse.microprofile.reactive.messaging.Message;
+import org.eclipse.microprofile.reactive.messaging.Outgoing;
+
+import io.smallrye.reactive.messaging.annotations.Blocking;
+
+@ApplicationScoped
+public class BeanReturningMessages {
+    private AtomicInteger count = new AtomicInteger();
+    private List<String> threads = new CopyOnWriteArrayList<>();
+
+    @Blocking
+    @Outgoing("infinite-producer-msg")
+    public Message<Integer> create() {
+        try {
+            Thread.sleep(200);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
```

Comment:
Can you add: Thread.currentThread().interrupt();

Reply:
(and remove the dump of the stack trace maybe)

Reply:
done